esp/esdllib/esdl_script.cpp

```diff
@@ -2548,7 +2658,7 @@ void processServiceAndMethodTransforms(IEsdlScriptContext * scriptCtx, std::init
     if (isEmptyString(reqtype))
         throw MakeStringException(ESDL_SCRIPT_Error, "ESDL script request name not set");
 
-    IEspContext *context = reinterpret_cast<IEspContext*>(scriptCtx->queryEspContext());
+    IEspContext *context = reinterpret_cast<IEsdlScriptContext*>(scriptCtx->queryUsageContext())->espContext();
```

Comment:
Why is reinterpret_cast<IEsdlScriptContext*>(scriptCtx->queryUsageContext()) necessary?  You already have an IEsdlScriptContext, get the pointer back as a void *, and then recast back to a IEsdlScriptContext?

Reply:
It isn't.